util/db/dbutil_test.go

```diff
@@ -238,6 +238,9 @@ func cleanupSqliteDb(t *testing.T, path string) {
 
 func TestDBConcurrencyRW(t *testing.T) {
 	partitiontest.PartitionTest(t)
+	if testing.Short() {
+		t.Skip()
```

Comment:
@algoidan Optional - If you feel it's worthwhile, I propose adding an explanation in `t.Skip("...")` (or inline comment) explaining why we think it's reasonable to skip during short test runs.  I think it will help future maintainers judge how to treat the test.

If you prefer as is, feel welcomed to resolve as is.

Aside:  I'm holding off on providing approval because I'm far enough removed from this part of the repo that I don't feel comfortable approving the change to nightly builds.

Reply:
I added a comment for this. Please let me know if that is a reasonable explanation.

Reply:
@algoidan Resolving - Thanks for adding a note.

Keep in mind, I _won't_ add approval because I don't consider myself knowledgeable enough to make a call here.